packages/components/src/components/bal-tabs/bal-tabs.tsx

```diff
@@ -342,11 +342,6 @@ export class Tabs
     if (activeTabs.length > 0) {
       const firstActiveTab = activeTabs[0]
       this.value = firstActiveTab.value
-    } else {
```

Comment:
- [ ] introduce new prop to control that new feature `optionalTabSelection `, so it is non breaking